charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -7,16 +7,36 @@ metadata:
     {{- include "newrelic.common.labels" . | nindent 4 }}
 data:
   daemonset-config.yaml: |
-    {{- with include "nrKubernetesOtel.daemonset.configMap.config" . }}
-    {{- . | nindent 4 }}
+    {{- if .Values.otel }}
+    receivers:
+      {{- include "daemonset-receivers" . | nindent 6 }}
+
+    processors:
+      {{- include "daemonset-processors" . | nindent 6 }}
+      {{- include "shared-processors" . | nindent 6 }}
+
+    exporters:
+      {{- include "shared-exporters" . | nindent 6 }}
+
+    connectors:
+      {{- include "daemonset-connector" . | nindent 6 }}
+
+    service:
+      {{- if include "newrelic.common.verboseLog" . }}
+      telemetry:
+        logs:
+          level: "debug"
+      {{- end }}
+      pipelines:
```

Comment:
Blocker.
It looks like we're not grabbing the "Service" configs from the yaml.
These sections ^ define all of the components but if we don't reference them in the service.pipelines these components won't do anything.